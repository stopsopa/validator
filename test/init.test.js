'use strict';

const add = require('../');

it('addaddaddaddaddaddaddaddaddadd', () => {
    expect(add(50, 21)).toMatchSnapshot();
});
