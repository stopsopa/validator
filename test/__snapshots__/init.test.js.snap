// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addaddaddaddaddaddaddaddaddadd 1`] = `71`;
